capstone/src/main/resources/static/event.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title> Create an Event </title>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  
+  <body>
+
+    <div class="menu">
+      <nav>
+        <a href="map.html"> Campus Map </a>
+      </nav>
+    </div>    
+
+    <h1>Create an Event </h1>
+
+    <div class="event-form">
+      <form id="form" method="POST">
+
+        <label for="organizationID"> OrganizationID: </label>
+        <input type="number" id="organizationID" name="organizationID"> 
+        <br><br>
+
+        <label for="organizationName"> Organization Name: </label>
+        <input type="text" id="organizationName" name="organizationName">
+        <br><br>
+
+        <label for="university"> University: </label>
+        <input type="text" id="university" name="university">
+        <br><br>
+
+        <label for="eventTitle"> Event Title: </label>
+        <input type="text" id="eventTitle" name="eventTitle">
+        <br><br>
+
+        <label for="eventTime"> Event Time: </label>
+        <input type="time" id="eventTime" name="eventTime">
+        <br><br>
+
+        <label for="eventLocation"> Event Location: </label> <br>
+        <input type="number" id="eventLat" name="eventLat" placeholder="Event Latitude"> <br>
+        <input type="number" id="eventLng" name="eventLng" placeholder="Event Longitude">
+        <br><br>
+
+        <label for="eventDescription"> Event Description: </label> <br>
+        <textarea id="eventDescription" name="eventDescription" rows="6" cols="60"> </textarea>
+        <br><br>
+
+        <label for="eventFilters"> Event Filters: </label>
+        <br>
+          <form>
+            <input type="checkbox" id="filter1" name="filter1" value="Disability Accomodations">
+            <label for="filter1"> Disability Accomodations </label>
+            <br>
```

Comment:
I think after our pod meeting last week we decided to not include this filter.

Reply:
Also, we have the indoor/outdoor status, activity level, and eco-conscious status. It's not super important to add them right now though!

Reply:
yeah, I can remove it, I was just putting binary answers into the form just to make sure it looked decent